### ISSUE TITLE
lack of instructions how to build jwt tool

### ISSUE THREAD

**Issue by greenpau** [dgrijalva/jwt-go#318]
There are no instructions how to build `jwt` tool. Perhaps, it is worth adding something like this.

```
cd cmd/jwt
go build -o $GOPATH/bin/jwt
```

**Comment by greenpau** [comment]
I also found a reference to it: https://github.com/go-swagger/go-swagger/tree/master/examples/composed-auth#prerequisites

```
go install github.com/dgrijalva/jwt-go/cmd/jwt
```

**Comment by dgrijalva** [comment]
Good point. I forgot that was in there. I'll make a note to add some documentation for the next release.
